net/utils.go

```diff
@@ -6,6 +6,8 @@ import (
 	"net"
 	"strings"
 	"time"
+
+	"github.com/mr-tron/base58/base58"
```

Comment:
I was curious to see if you'd used my base58 library: https://github.com/jimeh/go-base58

I was disappointed you hadn't, then I looked at the code of mr-tron's library, and went "oh, ok, this is fancy, nvm, my lib is shit" :)